### ISSUE TITLE
Add formatted multiple babies repeat group values to Pregnancy Outcome Event

### ISSUE THREAD

**Issue by allan-on** [opensrp/opensrp-client-chw#1974]
Following conversations with the Canopy team on [OpenSRP to Canopy connector handling of repeat groups](https://github.com/onaio/canopy/issues/2512),  the formatted repeat group values for the multiple babies recorded in a pregnancy outcome form should be added to the Event. 
The values will be in the following sample format: 
`"hhrepeat": [{ "hhrepeat/dob": "2020-07-16", "hhrepeat/gender": "1"... }, { "hhrepeat/dob": "2020-07-07", "hhrepeat/gender": "1".. }...]`
The repeat group can be processed and formatted using a Utility derived from [org.smartregister.reveal.util.Utils#buildRepeatingGroup](https://github.com/opensrp/opensrp-client-reveal/blob/0bbb9550a4cb150f7d4ba669df94fc5522da6f6c/opensrp-reveal/src/main/java/org/smartregister/reveal/util/Utils.java#L507)
The result will be written to the Event JSON on form submission

---
See https://github.com/onaio/canopy/issues/2512#issuecomment-987852853 for details

**Comment by alepietrobon** [comment]
what's the status of this, @allan-on ?

**Comment by allan-on** [comment]
@alepietrobon Work on this didn't start as it was scheduled prior to the break for the holidays.
I'll be working on this issue, this week.

**Comment by alepietrobon** [comment]
yeah i thought so... plz tell me it's top of your priority list now? :)

**Comment by allan-on** [comment]
Yes, it is 👍🏾

**Comment by allan-on** [comment]
- This is now done in the `chw-anc` PR mentioned (pending merging but available for QA through an alpha snapshot version) 
- The Pregnancy Outcome Event will have an additional `child_repeat_group_values_list` Obs that contains the multiple babies' details JSON in the **`values`** array
``` json 
{
...
    "eventType": "Pregnancy Outcome",
...
    "obs": [
        ...
        {
            "fieldCode": "child_repeat_group_values_list",
            "fieldDataType": "repeatvalueslist",
            "fieldType": "concept",
            "formSubmissionField": "repeatvalueslist",
            "saveObsAsArray": false,
            "values": [
                "[{\"lbw\":\"Yes\",\"disabilities\":\"No\",\"unique_id\":\"3db89ed457654e9db950aa787c083987\",\"same_as_fam_name\":\"true\",\"essential_newborn_care\":\"Cordon ombilical traité avec une solution de chlorhexidine\",\"early_bf_1hr\":\"Yes\"},{\"lbw\":\"Yes\",\"disabilities\":\"No\",\"unique_id\":\"37be1cdb126442499597c3af02e131d7\",\"same_as_fam_name\":\"true\",\"essential_newborn_care\":\"Aucun\",\"early_bf_1hr\":\"Yes\"}]"
            ]
        }
    ],
   ...
    "clientDatabaseVersion": 6,
    "dateCreated": "2022-01-12T11:40:24.910Z",
    "type": "Event"
}```
